### PR TITLE
add widget

### PR DESCRIPTION
**Pull request by hbakhtiyor**

### CODE REVIEW

File: src/apps.json [hunk 1]
```diff
@@ -9950,6 +9950,14 @@
 			"icon": "xui.png",
 			"script": "[^a-z]xui.*\\.js",
 			"website": "http://xuijs.com"
+		},
+		"Kontaktify": {
+			"cats": [
+				"5"
+			],
+			"icon": "Kontaktify.png",
+			"script": "//(?:www\\.)?kontaktify\\.com/embed.js(?:/)?\\?[A-Za-z0-9+/=]*",
```
Comment (jvoisin) [inline src/apps.json]: I think that you can drop the `(?:/)?\\?[A-Za-z0-9+/=]*` ending part of the regexp, since it doesn't improve the fingerprinting.

**Comment by hbakhtiyor** [comment]
@jvoisin done

**Comment by hbakhtiyor** [comment]
what's the reason not to merge it?

**Comment by craiglondon** [comment]
Jvosin doesn't do merges into master. Aliasio does

File: src/apps.json [hunk 2]
```diff
@@ -9950,6 +9950,14 @@
 			"icon": "xui.png",
 			"script": "[^a-z]xui.*\\.js",
 			"website": "http://xuijs.com"
+		},
+		"Kontaktify": {
+			"cats": [
+				"5"
+			],
+			"icon": "Kontaktify.png",
+			"script": "//(?:www\\.)?kontaktify\\.com/embed.js",
```
Comment (jvoisin) [inline src/apps.json]: Please add `^` in front of the regexp to speed it a bit if possible.

Comment (hbakhtiyor) [inline src/apps.json]: why? it maybe start with "http://..." or "https://..." or "//..."

Comment (jvoisin) [inline src/apps.json]: Hence my "if possible".
A lot of CND are giving url starting with `//`, so everyone uses them, maybe kontactify does the same.

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**